bin/bridge-client/src/constants.rs

```diff
@@ -4,5 +4,6 @@ pub(super) const DEFAULT_RPC_PORT: u32 = 4781; // first 4 digits in the sha256 o
 
 pub(super) const DEFAULT_RPC_HOST: &str = "127.0.0.1";
 
+pub(super) const DEFAULT_DUTY_TIMEOUT_DURATION: u64 = 65;
```

Comment:
Should increase this default value to at least 600. I know this is a temporary measure but this is to avoid breaking stuff if we ever deploy this change our signet.